qt/src/SourceManager.cc

```diff
@@ -44,6 +54,34 @@ QDataStream& operator>>(QDataStream& ds, Source*&) {
 	return ds;
 }
 
+Source::Source(const QString& _path, const QString& _displayname, bool _isTemp)
+				: path(_path), displayname(_displayname), isTemp(_isTemp) {
+	// Check is input PDF file encrypted
```

Comment:
// Check whether input PDF file is encrypted

Reply:
fixed.